Logos/cisco-logo-72px.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <desc>Created with Sketch.</desc>
```

Comment:
Can you please remove the attribute <desc> completely. Its invalid attribute.

Reply:
Completed

Reply:
Can you please remove  "<desc>Created with Sketch.</desc>"?

Reply:
Remove desc element ( desc-Created with Sketch-desc).